src/Panel/panel.scss

```diff
@@ -92,6 +92,14 @@ $content-title-color: $brc-darkGrey !default;
     border-left: 1px solid $border-color;
   }
 
+  &.floating {
```

Comment:
:(
Is this PR going to be "breaking" again? didn't you fixed the styles in the last one?